lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1323,11 +1364,22 @@ func (w *WalletKit) sweepNewInput(op *wire.OutPoint, currentHeight uint32,
 			"transaction")
 	}
 
-	// If there's no budget set, use the default value.
+	// TODO(ziggie): The budget value should ideally only be set for CPFP
+	// requests because for RBF requests we should have already registered
+	// the input including the budget value in the first place. However it
+	// might not be set and then depending on the deadline delta fee
+	// estimations might become too aggressive. So need to evalute whether
+	// we set a default value here, make it configurable or fail request
```

Comment:
To make what configurable?

Reply:
I thought about making the `DefaultBudgetRatio` configurable here at least ? But that's just one of the ideas.

Reply:
I think that's already configurable

Reply:
hmm, tho here the default value of 0.5 is used.

Reply:
yeah right, forgot we need to pass that config value here